misc/docs-linter/util.go

```diff
@@ -0,0 +1,107 @@
+package main
```

Comment:
These funcs should be in the same file as where they're called

Reply:
I've moved all functions to the `main.go` file in this commit: [3172ee8](https://github.com/gnolang/gno/pull/2125/commits/3172ee8c82596e6b867e88964a18977b89cfb87f)